pull.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+import requests
+import plotly.offline as py
+import plotly.graph_objs as go
+import io
+
+index = ['MU','GE','AMD','BAC']
+
+
+def main():
+    stocks = pd.DataFrame()
+    i = 0
+    while i < len(index):
+        r = requests.get('https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=' + str(
+            index[i]) + '&datatype=csv&apikey=603MDIJGG9TGNV60').content
+        df = pd.read_csv(io.StringIO(r.decode()))
```

Comment:
Can you explain what the ioStringIO module is doing here? Did passing in r.content.decode() directly not work?

Reply:
I'm probably dumb, but I wasn't able to get r.content.decode to work directly, but I was able to get it to work by using io like that